vtex/omnishipping
# CHK-88: Seller name

#### What is the purpose of this pull request?


This PR changes the omnishipping in order to show the seller's name instead of his id.

#### How should this be manually tested?

- Use this [workspace](http://jeff--promart.myvtex.com/checkout/cart/add/?sku=104128&qty=1&seller=1&sc=2&sku=9992524&qty=1&seller=oechsle&sc=2)
- Proceed to checkout
- Enter a new user
- Use `12345678` as the ID and `11111111` as the phone number
- On delivery use `Lima / Lima / Lima`
- Change to pickup
- You should see the sellers info as in the image below

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/12574426/88681706-3d609d80-d0c8-11ea-9dc0-f390eb8290d9.png)


#### Types of changes
